db/monitor/ACR06B1.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0"?>
+<monitor name="Acer Nitro XV273K" init="standard">
+	<caps add="(prot(monitor)type(lcd)model(XV273K)cmds(01 02 03 07 0C E3 F3)vcp(02 04 05 08 0B 0C 10 12 14(05 06 08 0B 0C) 16 18 1A 52 54(00 01) 59 5A 5B 5C 5D 5E 60(03 11 0F 10) 62 9B 9C 9D 9E 9F A0 AC AE B6 C0 C6 C8 C9 CC(01 02 03 04 05 06 08 09 0A 0C 0D 0E 14 16 1E)D6(01 04 05) DF E3 E5 E6 E7(00 01 02) E8(00 01 02) )mswhql(1)asset_eep(32)mccs_ver(2.1))" />
+
+	<controls>
+		<control id="brightness" address="0x10"/>
+		<control id="contrast" address="0x12"/>
+		<control id="red" address="0x16"/>
+		<control id="green" address="0x18"/>
+		<control id="blue" address="0x1A"/>
+
+		<control id="colorpreset" address="0x14">
+			<value id="warm" value="5"/>
+			<value id="normal" value="6"/>
+			<value id="cool" value="8"/>
+			<value id="user" value="11"/>
+			<value id="bluelight" value="12"/>
+		</control>
+
+		<!-- Black Boost doesn't seem to be working; only 0/10 and 1/10 can be set AFAICT. -->
+
+		<control id="redhue" address="0x9b"/>
+		<control id="greenhue" address="0x9d"/>
+		<control id="bluehue" address="0x9f"/>
+		<control id="yellowhue" address="0x9c"/>
+		<control id="magentahue" address="0xa0"/>
+		<control id="cyanhue" address="0x9e"/>
+
+		<control id="redsaturate" address="0x59"/>
+		<control id="greensaturate" address="0x5b"/>
+		<control id="bluesaturate" address="0x5d"/>
+		<control id="yellowsaturate" address="0x5a"/>
+		<control id="magentasaturate" address="0x5e"/>
+		<control id="cyansaturate" address="0x5c"/>
+
+		<control id="inputsource" address="0x60">
+			<value id="dp1" value="15"/>
+			<value id="dp2" value="16"/>
+			<value id="hdmi1" value="17"/>
+			<!-- HDMI 1 can be selected, but doesn't seem to register properly. When I hooked up my computer via both DP and HDMI and selected HDMI 1, the DDC Control GUI would still show the DP input as enabled. -->
+			<!-- HDMI 2 doesn't seem to be selectable at all. It's not represented by any value in the range 1–32 at least. -->
+		</control>
+
+		<control id="audiospeakervolume" address="0x62"/>
+
+		<control id="language" type="list" address="0xcc">
+			<value id="chinese_tw" value="1"/>
+			<value id="english" value="2"/>
+			<value id="french" value="3"/>
+			<value id="german" value="4"/>
+			<value id="italian" value="5"/>
+			<value id="japanese" value="6"/>
+			<value id="portuguese" value="8"/>
+			<value id="russian" value="9"/>
+			<value id="spanish" value="10"/>
+			<value id="turkish" value="12"/>
+			<value id="chinese" value="13"/>
+			<value id="brazilian" value="14"/>
+			<value id="dutch" value="20"/>
+			<value id="finnish" value="22"/>
+			<value id="polish" value="30"/>
+		</control>
+
+		<control id="dpms" address="0xd6">
+			<value id="on" value="1"/>
+			<value id="standby" value="4"/>
+		</control>
+
+		<control id="defaults" address="0x04" delay="2000"/>
+		<control id="defaultluma" address="0x05" delay="2000"/>
+		<control id="defaultcolor" address="0x08" delay="2000"/>
```

Comment:
```suggestion
```

These are same as in standard VESA profile. Please, remove these and other non-list controls, too.